lib/views/settings.dart

```diff
@@ -25,7 +27,7 @@ class _SettingsState extends State<Settings> {
   final CurrentUserPresenter currentUserPresenter;
   int credits = 0;
   int engCredits = 0;
-  bool isMetricsEnabled = false;
```

Comment:
Why true? What If popup fails? Should rather be false by default and then be set.

Reply:
That's not really a default value. Just the initialize, but I have changed it...

A few lines later it is overwritten each time with the user settings and those can't be null at this time. But you are right - we never now...